### DISCUSSION TITLE
Тепловая карта

### DISCUSSION THREAD

**Discussion by klimashkin** [PastVu/pastvu#12] [Ideas / Идеи и предложения]
В будущем будет интересно посмотреть распределение загруженных фотографий по карте с помощью тепловой карты.
Например: 
https://google-developers.appspot.com/maps/documentation/javascript/examples/layer-heatmap

http://sunng87.github.com/heatcanvas/leaflet.html

**Comment by Duche3d** [comment]
Ну наверное как дополнительный слой в будущем можно было бы добавить. Еслиб это еще как-то можно было объединить с информацией по времени - это было бы полезно.

**Comment by Duche3d** [comment]
или может по рейтингу просмотра фоток - это может оказаться весьма интересно, если например окажется, что места наибольшего скопления фото при этом не являются местами, фото по которым люди чаще всего смотрят

Jan 14, 2013, в 12:39 PM, klimashkin notifications@github.com написал(а):

**Comment by paul-k-pastvu** [comment]
@klimashkin, ты можешь пояснить, зачем это может быть нужно/интересно? Чтобы что?

**Reply by klimashkin** [reply]
Чтобы видеть очаги наибольшей привязки фотографий. Сейчас это в принципе очевидно, но в будущем будет больше центров. Еще было бы интересно сравнивать это во времени

**Reply by paul-k-pastvu** [reply]
И фильтровать по юзерам?

**Reply by yuriy-zhilovets2** [reply]
Тепловой картой можно заменить показ кластеров при большом масштабе. Квадратики с надписью "2000" довольно бесполезны и только заслоняют карту.
